src/command/output.rs

```diff
@@ -984,59 +983,74 @@ mod tests {
-        );
+        let target_url =
+            "https://studio.apollographql.com/graph/my-graph/composition/big-hash?variant=current"
+                .to_string();
+        let check_response = CheckWorkflowResponse {
+            default_target_url: target_url.clone(),
+            maybe_core_schema_modified: Some(false),
+            maybe_operations_response: Some(OperationCheckResponse::try_new(
+                CheckTaskStatus::FAILED,
+                Some(target_url),
+                10,
+                vec![
+                    SchemaChange {
+                        code: "SOMETHING_HAPPENED".to_string(),
+                        description: "beeg yoshi".to_string(),
+                        severity: ChangeSeverity::FAIL,
+                    },
+                    SchemaChange {
+                        code: "WOW".to_string(),
+                        description: "that was so cool".to_string(),
+                        severity: ChangeSeverity::FAIL,
+                    },
+                ],
+                ChangeSeverity::FAIL,
+            )),
+            maybe_lint_response: None,
+            maybe_downstream_response: None,
+        };
 
-        if let Err(operation_check_failure) = check_response {
-            let actual_json: JsonOutput = RoverError::new(operation_check_failure).into();
-            let expected_json = json!(
-            {
-                "json_version": "1",
-                "data": {
-                    "target_url": "https://studio.apollographql.com/graph/my-graph/composition/big-hash?variant=current",
-                    "operation_check_count": 10,
-                    "result": "FAIL",
-                    "changes": [
-                        {
-                            "code": "SOMETHING_HAPPENED",
-                            "description": "beeg yoshi",
-                            "severity": "FAIL"
-                        },
-                        {
-                            "code": "WOW",
-                            "description": "that was so cool",
-                            "severity": "FAIL"
-                        },
-                    ],
-                    "failure_count": 2,
-                    "success": false,
-                    "core_schema_modified": false,
-                },
-                "error": {
-                    "message": "This operation check has encountered 2 schema changes that would break operations from existing client traffic.",
-                    "code": "E030",
-                }
-            });
-            assert_json_eq!(expected_json, actual_json);
-        } else {
-            panic!("The shape of this response should return a RoverClientError")
-        }
+        let actual_json: JsonOutput = RoverError::new(RoverClientError::CheckWorkflowFailure {
+            graph_ref,
+            check_response,
+        })
+        .into();
+        let expected_json = json!(
+        {
+            "json_version": "1",
```

Comment:
since this is a breaking change to operation check output, we will need to bump this `json_version` to `2` (only for the check commands) so that folks handling rover output programatically can react to the change.

Reply:
we should also add some JSON unit tests for responses that _do_ include lint check tasks

Reply:
Good catch. Is there a preferred way of updating the `json_version` metadata for specific rover outputs?

Reply:
lint output test added in https://github.com/apollographql/rover/pull/1637/commits/ed8e862789dfeb51f8ccc9a3c26a7b3432807aec (along with a few other changes).

Reply:
we've never bumped a json version up until this point, but the entry point for it is in this function:

```rust
    pub(crate) fn get_json_version(&self) -> JsonVersion {
        JsonVersion::default()
    }
```

we'll want to change this to be something like this:

```rust
    pub(crate) fn get_json_version(&self) -> JsonVersion {
        match &self {,
			Self::CheckResponse(_) => JsonVersion::Two,
        	_ => JsonVersion::default()
        }
    }
```

(you'll need to add `JsonVersion::Two` to the `JsonVersion` enum as well)

Reply:
Added your suggestion in [4d8f36e](https://github.com/apollographql/rover/pull/1637/commits/4d8f36e436f77479b8b6b211cc66bd0ba8d3af87) and updated checks related tests to reflect the `json_version` bump to `2`.